### PR TITLE
fixed helm chart test

### PR DESCRIPTION
**Pull request by raffaelespazzoli**
removed nginx controller

### CODE REVIEW

**Comment by raffaelespazzoli** [comment]
so I made a mess, somehow I pushed to upstream instead of origin (I think because after the repo transfer origin didn't actually exist anymore), so after this merge we should also delete this branch. @sabre1041

File: config/local-development/tilt/replace-image.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ spec:
     spec:
       containers:
         - name: manager
-          image: quay.io/raffaelespazzoli/gitwebhook-operator:latest
+          image: quay.io/gitwebhook-operator/gitwebhook-operator:latest
```
Comment (sabre1041) [inline config/local-development/tilt/replace-image.yaml:11]:
```suggestion
          image: quay.io/redhat-cop/gitwebhook-operator:latest
```

**Review by sabre1041 — CHANGES_REQUESTED** [review]
Small change

**Comment by raffaelespazzoli** [comment]
suggestion commited

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**